opengever/maintenance/scripts/rename_deployment.py

```diff
@@ -83,6 +98,64 @@ def set_new_org_unit_title(plone, options):
         org_unit.title = new_ou_title
 
 
+def set_new_admin_unit_abbreviation(plone, options):
+    new_au_abbr = options.new_au_abbr.decode('utf-8')
+
+    admin_unit = get_current_admin_unit()
+
+    logger.info('Existing AdminUnit abbreviation: %s' % admin_unit.abbreviation)
+    logger.info('Setting AdminUnit abbreviation: %s\n' % new_au_abbr)
+
+    if not options.dryrun:
+        admin_unit.abbreviation = new_au_abbr
+
+
+def reindex_objects(plone, options):
+
+    catalog = api.portal.get_tool('portal_catalog')
+    brains = catalog.unrestrictedSearchResults()
+
+    for brain in brains:
+        obj = brain.getObject()
+        attrs_to_index = []
+
+        if options.new_au_abbr:
+            # Reindex data dervived from AdminUnit abbreviation:
+            # - Catalog indexes / metadata
+            # - Solr fields
+            # - SQL (for tasks)
+
+            # (Almost) everything has at least a reference number
+            attrs_to_index.append('reference')
+
+            # Update the 'reference' column in SQL
+            logger.info('Syncing task %r to SQL' % obj)
+            if ITask.providedBy(obj) and not options.dryrun:
+                sync_task(obj, None)
```

Comment:
True, but since the `modified` date on the Plone object doesn't change, nothing different gets propagated to the SQL `modified`. At least that's what I was obverving locally, SQL `modified` stays the same even after syncing.

Reply:
Ah perfect 👍